Dwarf_Guilds.cat

```diff
@@ -1,874 +1,604 @@
-          </entryLinks>
-        </selectionEntryGroup>
-      </selectionEntryGroups>
+      <categoryLinks>
+        <categoryLink id="505d-524d-acb0-7beb" name="New CategoryLink" hidden="false" targetId="4b4e-fbe0-5211-ae65" primary="true"/>
+      </categoryLinks>
       <entryLinks>
-        <entryLink id="0e01-0832-3a9e-731e" name="CCW (A1)" hidden="false" collective="false" import="true" targetId="b521-2bfc-867e-4ea1" type="selectionEntry">
-          <constraints>
-            <constraint field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="8925-d515-dc9e-38a2" type="max"/>
-            <constraint field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="45e7-cf60-04b8-3f19" type="min"/>
-          </constraints>
-        </entryLink>
+        <entryLink id="3e1f-f4cc-3c32-ff6a" name="Upgrade List A" hidden="false" collective="false" import="true" targetId="be30-aec4-bcc3-9830" type="selectionEntryGroup"/>
+        <entryLink id="58a1-c282-de85-5dc6" name="Upgrade List B" hidden="false" collective="false" import="true" targetId="05aa-8e12-6eb7-3c0c" type="selectionEntryGroup"/>
       </entryLinks>
       <costs>
-        <cost name="pts" typeId="567f-6468-66c6-2ea2" value="0.0"/>
+        <cost name="pts" typeId="567f-6468-66c6-2ea2" value="55.0"/>
       </costs>
     </selectionEntry>
-    <selectionEntry id="8d0a-3e99-04fa-29c3" name="Dwarf Warrior (Leader)" hidden="false" collective="false" import="true" type="upgrade">
+    <selectionEntry id="9d6b-f88e-fb8d-48ce" name="Rune Master" hidden="false" collective="false" import="true" type="unit">
+      <profiles>
+        <profile id="ea43-cea2-04ec-6f4c" name="Rune Master" hidden="false" typeId="a8fa-e9ce-c38a-c73e" typeName="Unit">
+          <characteristics>
+            <characteristic name="Quality" typeId="c619-fc26-5d0b-187d">3+</characteristic>
+            <characteristic name="Defense" typeId="5564-b1cb-27d0-1af7">3+</characteristic>
+            <characteristic name="Special Rules" typeId="6039-8041-2416-3f32">Hero, Psychic(1), Slow, Tough(3)</characteristic>
+          </characteristics>
+        </profile>
+      </profiles>
+      <infoLinks>
+        <infoLink id="a2b7-01c5-2943-a786" name="Psychic(X)" hidden="false" targetId="ba47-b43b-18f8-97c1" type="rule"/>
+        <infoLink id="02cc-d727-5349-f54e" name="Hero" hidden="false" targetId="5065-c3a4-a9cf-db27" type="rule"/>
+        <infoLink id="4332-8fd4-45d8-e4a1" name="Slow" hidden="false" targetId="394b-1b64-d270-f49e" type="rule"/>
+        <infoLink id="7cc2-988b-a607-3252" name="Tough(X)" hidden="false" targetId="b9d3-4d17-007c-22cb" type="rule"/>
+        <infoLink id="d049-7ce8-a45b-8b64" name="Psychic Spells (Dwarf Guilds)" hidden="false" targetId="b0ca-cf90-a910-15e7" type="rule"/>
```

Comment:
@nicolasconnault with this update I tried something new. Instead of a "Show Spells" entry toggle, I wrote up the spell descriptions in an army-specific rule and then that rule is referenced whenever Psychic is. This means that users will always get the spell readout in their Special Rules section without having to select anything, but also prevents them from filling their roster with a ton of duplication if they select it multiple times.

I like that newer approach and plan to use it in further updates, but if you have differing thoughts just let me know.

Reply:
Oh I think it's great, go ahead, much better than before :)